### DISCUSSION TITLE
HW_07 Clarification Problem 2

### DISCUSSION THREAD

**Discussion by RyanW-UConn** [cooperrc/advanced-dynamics#48] [Q&A]
Hello,
I wasn't able to attend the live session Tuesday, and that probably would have been an easier place to ask - but if anyone has any advice that would be great. I did watch the session recording, but I don't think my question below was directly addressed.

My question is simply what is the deliverable for Problem 2 in HW_07? I've gone over the section a few times, and best I can tell is that the deliverables are a plot of $`{R}^3_x`$ (slider block displacement) vs Crank angle to match the figure from the textbook. But this is already done in the code with this line given: `plt.plot(q[2,:],q[6,:])`. Then later in the section on Problem 2 there is a prompt to plot $`\dot{\theta}^2,~\ddot{\theta}^2,~\dot{R}^3_x, \ddot{R}^3_x`$, to match graphs from textbook, I've plotted these as well - but I feel like I might be missing something? Is the goal just to make some plots using the given functions here while the more in-depth task of changing those functions is left for Problem 3?

Thanks,
Ryan

**Comment by RebeccaVillanueva** [comment]
I have this same exact question, so I'm just commenting now to stay in the loop haha

**Comment by travismalave** [comment]
Based on the discussed had in the Q&A, my understanding is the main deliverables for HW_07 are:
a) Problem 1 = solve for q(t=0)
b) Problem 2 = replicate the crank angle vs. XXX plots (I think this is more for the velocity and acceleration plots and the "Problem 2" header is in a weird spot - you can simply just uncomment the plot code for the slider block and call it a day)
c) Problem 3 = recreate the solution using the provided "def" code to account for the new constraint and create the remaining plots. 

The only unknown to me is the animation section since it seems like its all set up but the animation seems off. Not sure what's needed there if anything at all.
